packages/admin-ui/src/common/types.ts

```diff
@@ -441,6 +448,14 @@ export type InstalledPackageData = Pick<
   containers: PackageContainer[];
 };
 
+export interface PortsScanResponse {
+  host: string; //"83.32.61.31"
+  ports: {
+    open: number[]; // [8092,4001]
```

Comment:
Even if we only scan TCP, this information should be clear here. The array should be of type `{port: number, protocol: "TCP" | "UDP" }`

Reply:
It should return an array of all the ports requested with the status as a property, instead of sorted in two arrays.
```ts
ports: {
  port: number;
  protocol: "TCP" | "UDP";
  status: "open" | "closed" | "error";
  error?: string;
}[]
```